notebooks/tutorials/enclaves/Enclave-single-notebook-DO-DS.ipynb

```diff
@@ -13,6 +13,7 @@
    ]
   },
   {
+   "attachments": {},
```

Comment:
Q: Are both notebooks different in tutorials/enclaves?
I noticed similar testing.
We could keep one and delete the other?

Reply:
one contains the high side and low side, the other does not

Reply:
From a testing perspective they have the same coverage. From a tutorial perspective I think they are different things